src/components/x-clock/x-clock-react.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 import {Component} from "@micro-frontends/core";
 import {renderReact} from "@micro-frontends/framework";
 
-export class XClockReact extends Component {
+export class XClockReact extends Component<undefined, undefined> {
```

Comment:
Component<undefined, undefined> would it be default values?

Reply:
fixed to default values